Tutorials/seL4Kernel/setting-up.md

```diff
@@ -0,0 +1,237 @@
+---
+toc: true
+layout: tutorial
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Setting up your machine
+**Overview**
+- Set up your machine - install dependencies required to run seL4``
+- Run seL4test on a simulator
+- Gain awareness of terminology used for seL4
+
+
+
+## Docker
+To compile and use seL4, it is recommended that you use Docker to isolate the dependencies from your machine.
```

Comment:
If we are recommending docker, we don't really need the rest of the page apart from `repo` (although I don't know if the virtual machine tutorials actually work under docker). We either should remove the docker section or add some text that makes clear that docker and the rest are two alternatives, and that you need only one of them.

Reply:
I actually removed everything but the repo and docker sections, because there is a whole other page on dependencies https://docs.sel4.systems/projects/buildsystem/host-dependencies.html, should someone need them.

I couldn't get of the virtual machine tutorials working on my silicon mac (but I don't think it's related to Docker at all).

Adding issues here for completeness:

"CAmkES VM: Adding a Linux Guest" uses KVM and I would have needed an alternative for mac. 

"CAmkES VM: Cross VM Connectors" breaks using `ninja`.

```
sh.ErrorReturnCode_1: 

  RAN: /usr/bin/cmake -G Ninja -DTUT_BOARD=pc -DTUT_ARCH=x86_64 -DFORCE_IOMMU=ON -DTUTORIAL_DIR=camkes-vm-crossvmke7jqji2 -C ../projec

  STDOUT:


  STDERR:
```

Should I add a note to these tutorials saying the instructions are for Linux only?

Reply:
Yes, that would be good. We've already had some questions from people who were trying to use other platforms.